security/4.0/_index.md

```diff
@@ -26,18 +23,14 @@ The release splits focuses on evolving the API in various ways, and specifically
 
 * [Jakarta Security 4.0 Release Record](https://projects.eclipse.org/projects/ee4j.security/releases/4.0)
 
-<!--
-The following can be uncommented and version information updated as they become available.
-
 * [Jakarta Security 4.0 Specification Document](./jakarta-security-spec-4.0.pdf) (PDF)
 * [Jakarta Security 4.0 Specification Document](./jakarta-security-spec-4.0.html) (HTML)
 * [Jakarta Security 4.0 Javadoc](./apidocs)
 * [Jakarta Security 4.0 TCK](https://download.eclipse.org/jakartaee/security/4.0/jakarta-security-tck-4.0.0.zip) ([sig](https://download.eclipse.org/jakartaee/security/4.0/jakarta-security-tck-4.0.0.zip.sig), [sha](https://download.eclipse.org/jakartaee/security/4.0/jakarta-security-tck-4.0.0.zip.sha256), [pub](https://raw.githubusercontent.com/jakartaee/specification-committee/master/jakartaee-spec-committee.pub))
--->
 
 # Compatible Implementations
 
-* Soteria (TBC)
+* Soteria 4.0.0
```

Comment:
A link to the project is quite usual even though the name is probably enough.

Reply:
Thanks for noticing. It is done!